ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -567,3 +567,36 @@ def get_blockpools_compression_status_from_storagesystem(self) -> tuple:
             self.validation_loc["storagecluster-blockpool-details-compress-status"]
         )
         return compression_status_blockpools_tab, compression_status_blockpools_details
+
+    def verify_odf_operator_in_installed_operator(self) -> bool:
+        """
+        Function to validate either ODF operator pod is up and running post ODF installation,
+        expectation is only ODF operator should be running in Installed operators tab other operators shouldn't be there
+        This function is only written for 4.9+ versions
+
+        Returns:
+        True: If only odf operator is present in the UI
+        False: If ocs operator is also present in the UI
```

Comment:
False is returned also when odf operator is `not` found, right?

Reply:
No, there has been cases where odf operator and ocs operator both are present in the installed operator, as seen in the bz [1994584](https://bugzilla.redhat.com/show_bug.cgi?id=1994584). Thus we checking that only odf operator is present and ocs operator should not be present post 4.9 versions of ocp